topological_navigation/package.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>topological_navigation</name>
   <version>2.4.0</version>
   <description>The topological_navigation package</description>
 
-
   <maintainer email="abinch@sagarobotics.com">Adam Binch</maintainer>
 
```

Comment:
I think we need to add you or somebody else from UoL as a maintained here @Iranaphor (you can also list me)